src/tests/ftest/telemetry/telemetry_pool_metrics.py

```diff
@@ -158,8 +158,13 @@ def test_telemetry_pool_metrics(self):
             self.ior_cmd.dfs_chunk.update(self.ior_cmd.transfer_size.value)
             self.run_ior_with_pool(
                 timeout=200, create_pool=False, create_cont=False)
-        except TestFail:
-            self.log.info("#ior command failed!")
+        except TestFail as error:
+            # NOTE DAOS-12946:  it is not possible to properly test the metrics as we do not know
+            # how much data have been transferred.  Thus we cancel the test, as the purpose of this
+            # test is not to check the ior command.
+            self.log.error(">>>>>> IOR command failed: %s", str(error))
+            self.log.info("------ Test canceled ------")
+            self.cancel("IOR command failed: {}".format(error))
```

Comment:
I wonder if the test should just fail here - `self.fail` - since if IOR fails that's probably a different problem that needs to be investigated. I commented on the ticket as well

Reply:
Good point, I will change the PR according to that.

- [ ] Fail on ior error